compiler+runtime/CMakeLists.txt

```diff
@@ -257,6 +257,8 @@ include(cmake/dependency/fmt.cmake)
 include(cmake/dependency/libzippp.cmake)
 
 find_package(OpenSSL REQUIRED COMPONENTS Crypto)
+set(Boost_USE_STATIC_LIBS ON)
+set(Boost_USE_STATIC_RUNTIME OFF)
```

Comment:
What's the static runtime? Will you please put it as a comment so I can also know in 6 months when I ask myself the same thing? :joy: